src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,195 @@
+.Navigation{
+    background-color: #99D2E9;
+    color: white;
+    font-family: 'Roboto', sans-serif;
+    line-height: 24px;
+    font-weight: 300;
+    height: 70px;
+    width: 100%;
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+}
+
+/* .Navigation > .Nav-btn {
+  display: none;
+} */
+
+.Navigation > .NavLinks {
+  align-self: flex-end;
+  font-size: 18px;
+  height: 100%;
```

Comment:
Please Remove this line that is when the nav links become Centered Horizontally
```suggestion

```
This..
![Screenshot from 2020-02-14 09-35-21](https://user-images.githubusercontent.com/18049550/74507433-72f43e80-4f0d-11ea-94fb-65e9f0854e76.png)

Not this
![Screenshot from 2020-02-14 09-36-59](https://user-images.githubusercontent.com/18049550/74507517-af279f00-4f0d-11ea-8f9b-4b48b1e72bfe.png)